src/nft/components/collection/CollectionNfts.css.ts

```diff
@@ -20,12 +20,6 @@ export const assetList = style([
       [`screen and (min-width: ${breakpoints.xl}px)`]: {
         gridTemplateColumns: 'repeat(auto-fill, minmax(calc(100%/4 - 16px), 1fr) )',
       },
-      [`screen and (min-width: ${breakpoints.xxl}px)`]: {
-        gridTemplateColumns: 'repeat(auto-fill, minmax(calc(100%/5 - 20px), 1fr) )',
-      },
-      [`screen and (min-width: ${breakpoints.xxxl}px)`]: {
-        gridTemplateColumns: 'repeat(auto-fill, minmax(calc(100%/7 - 20px), 1fr) )',
-      },
```

Comment:
we should still have these lines but replace the `20px` with `16px`

Reply:
ah, didn't catch the 5 and 7 -- thought this was just repetition. 

good catch tyvm